src/screens/Settings/ReportIssue/index.tsx

```diff
@@ -0,0 +1,156 @@
+import React, { memo, ReactElement, useState } from 'react';
+import axios from 'axios';
+import { View, StyleSheet, Platform } from 'react-native';
+import {
+	getBuildNumber,
+	getSystemVersion,
+	getVersion,
+} from 'react-native-device-info';
+import { Trans, useTranslation } from 'react-i18next';
+
+import { getNodeId, getNodeVersion } from '../../../utils/lightning';
+import { Text01S } from '../../../styles/text';
+import NavigationHeader from '../../../components/NavigationHeader';
+import GlowingBackground from '../../../components/GlowingBackground';
+import SafeAreaInset from '../../../components/SafeAreaInset';
+import type { SettingsScreenProps } from '../../../navigation/types';
+import LabeledInput from '../../../components/LabeledInput';
+import Button from '../../../components/Button';
+
+const ReportIssue = ({
+	navigation,
+}: SettingsScreenProps<'ReportIssue'>): ReactElement => {
+	const { t } = useTranslation('settings');
+	const [email, setEmail] = useState('');
+	const [message, setMessage] = useState('');
+	const [isLoading, setIsLoading] = useState(false);
+
+	const validateEmail = (emailText: string): boolean => {
+		if (emailText.indexOf('@') !== -1) {
+			const [beforeAt, afterAt] = emailText.split('@');
+			if (beforeAt.length > 0 && afterAt.length > 0) {
+				return true;
+			}
+		}
+		return false;
+	};
+
+	const SendRequest = async (): Promise<void> => {
+		try {
+			setIsLoading(true);
+			let ldkVersion = '';
+			let ldkNodeId = '';
+			const ldkVersionUser = await getNodeVersion();
+			const ldknodeIdUser = await getNodeId();
+
+			if (ldkVersionUser.isOk() && ldknodeIdUser.isOk()) {
+				ldkVersion += `ldk-${ldkVersionUser.value.ldk} c_bindings-${ldkVersionUser.value.c_bindings}`;
+				ldkNodeId += `${ldknodeIdUser.value}`;
+			}
+			await axios.post(`${process.env.API_FORM}`, {
+				email,
+				message,
+				platform: `${Platform.OS} ${getSystemVersion()}`,
+				version: `${getVersion()} (${getBuildNumber()})`,
+				ldkVersion: ldkVersion,
+				ldkNodeId: ldkNodeId,
+			});
+			navigation.navigate('ConfirmFormSent');
+			setEmail('');
+			setMessage('');
+			setIsLoading(false);
+		} catch (error) {
+			console.error('Error', error);
+			navigation.navigate('ErrorFormSent');
+			setEmail('');
+			setMessage('');
+			setIsLoading(false);
+		}
+	};
+
+	const isValid = validateEmail(email) && message;
+
+	return (
+		<GlowingBackground>
+			<SafeAreaInset type="top" />
+			<NavigationHeader
+				title={t('support.report')}
+				onClosePress={(): void => {
+					navigation.navigate('Wallet');
+				}}
+			/>
+			<View style={styles.content}>
+				<Text01S style={styles.text} color="gray1">
+					<Trans
+						t={t}
+						i18nKey="support.report_text"
+						components={{
+							easterEgg: <Text01S color="gray1" />,
+						}}
+					/>
+				</Text01S>
```

Comment:
Couple of unnecessary uses of `<Trans />` component, use the text directly like:

```suggestion
				<Text01S style={styles.text} color="gray1">
					{t('support.report_text')}
				</Text01S>
```